.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Describe your changes
+
+## Issue ticket if applicable
+
+<!-- Fix: https://github.com/jupyter/docker-stacks/issues/0 -->
+
+## Checklist (especially for first-time contributors)
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests
+- [ ] I will try not to use force-push to make the review process easier for reviewers
+
```

Comment:
What about adding:

- [ ] I have updated the documentation for significant changes